mewarpx/changelog.csv

```diff
@@ -28,7 +28,7 @@ Version, Physics version, Date,        List of changes
   Neumann, but the actual BC was hard coded to be Dirichlet with 0 V potential.
   This has now been fixed.
 "
-4.0.0, 2, 12/31/2021, "
+4.0.0, 2, 12/29/2021, "
```

Comment:
```suggestion
4.0.0, 2, 12/23/2021, "
```
This is the 4.0.0 date

Reply:
12/29/21 should go in line 2 of the file.

Reply:
I've definitely been updating these incorrectly the whole time. Every time I update the changelog the current version (5.0.0 in this case) remains `In progress` until there is a newer version ready for release at which point the (end)date for the current version is updated. Point being I've been treating it as a deprecation date not a release date.
Happy to switch it but just know that most of the old dates are wrong then.

Reply:
Ah, ok. I see why my previous practice confused you: Because I never knew when PR reviews were going to finish, I just typically left the newest version as "In Progress" always so I didn't have to remember to update it right before the PR merge. But then next time I altered the changelog, I would go to git history and figure out what the last version release date should have been and update it accordingly.

Let's use release date, since that's obviously more standard for newcomers. Since we have git history anyway, I don't care about updating previous ones though.